tests/unit/amo/components/TestAddonMoreInfo.js

```diff
@@ -139,17 +140,17 @@ describe(__filename, () => {
   });
 
   it('does not render a support link if none exists', () => {
-    const partialAddon = createInternalAddon(fakeAddon);
+    const partialAddon = createInternalAddonWithLang(fakeAddon);
     delete partialAddon.support_url;
     const root = render({ addon: partialAddon });
 
     expect(root.find('.AddonMoreInfo-support-link')).toHaveLength(0);
   });
 
   it('renders the support link of an add-on', () => {
-    const addon = createInternalAddon({
+    const addon = createInternalAddonWithLang({
       ...fakeAddon,
-      support_url: 'http://support.hampsterdance.com/',
+      support_url: createLocalizedString('http://support.hampsterdance.com/'),
```

Comment:
I haven't seen this function yet. I suppose it uses a default lang? How do we sync `createInternalAddonWithLang` and `createLocalizedString`? Maybe we could name these functions `createInternalAddonWithDefaultLang` and `createLocalizedStringWithDefaultLang`?

It's verbose but explicit. Other ideas welcomed, though.

Reply:
As discussed above (and I think as you noticed later), `createLocalizedString` does optionally accept a `lang` if we need to synchronize. Most of the time we don't, which makes the test code and its usage shorter. To me it's fine for a test to use both `createInternalAddonWithLang` and `createLocalizedString` without passing a `lang` to either, and assume that they are synchronized behind the scenes (which they are). Do you agree?

Reply:
I agree.